modules/mysql/read_replica.tf

```diff
@@ -41,6 +41,13 @@ resource "google_sql_database_instance" "replicas" {
     availability_type           = lookup(each.value, "availability_type", var.availability_type)
     deletion_protection_enabled = var.read_replica_deletion_protection_enabled
 
+    dynamic "backup_configuration" {
+      for_each = lookup(each.value, "backup_configuration") != null ? [lookup(each.value, "backup_configuration")] : []
```

Comment:
That makes sense actually, since omitting `default` in lookup is practically the same as using the plain `map[key]`. Not to mention it's deprecated but I guess old habits. :)

Thanks for pointing this out, will make the adjustment!